release-notes/major12/12.0.0/valtimo-frontend-libraries.md

```diff
@@ -4,15 +4,14 @@
 
 The following features were added:
 
-* **New feature1**
+* **Inline edit document definition**
 
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
+  Document definition can now be editted inline to create a new version.
 
-* **New feature2**
+* **Pending changes modal extension**
 
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
+  Pending changes modal has been extended to accept custom modals.
```

Comment:
Can you add a reason as to why people would want this? Maybe in the form of an example?